pkgs/by-name/ke/kexec-tools/package.nix

```diff
@@ -28,6 +28,11 @@ stdenv.mkDerivation rec {
       url = "https://raw.githubusercontent.com/void-linux/void-packages/6c1192cbf166698932030c2e3de71db1885a572d/srcpkgs/kexec-tools/patches/ppc64-elfv2.patch";
       sha256 = "19wzfwb0azm932v0vhywv4221818qmlmvdfwpvvpfyw4hjsc2s1l";
     })
+    # Fix for static builds
```

Comment:
Maybe note that this can be removed once the package is updated?

Reply:
added!